### ISSUE TITLE
Run fails on python 3.10

### ISSUE THREAD

**Issue by GoshaEgorian** [pyfa-org/Pyfa#2443]
## Bug Report

Pyfa fails to run on python 3.10

### Expected behavior:

Pyfa runs normally

### Actual behavior:

Error appear:
```
Traceback (most recent call last):
  File "/usr/local/Pyfa/pyfa.py", line 148, in <module>
    from gui.mainFrame import MainFrame
  File "/usr/local/Pyfa/gui/mainFrame.py", line 36, in <module>
    import gui.fitCommands as cmd
  File "/usr/local/Pyfa/gui/fitCommands/__init__.py", line 1, in <module>
    from .gui.booster.add import GuiAddBoosterCommand
  File "/usr/local/Pyfa/gui/fitCommands/gui/booster/add.py", line 6, in <module>
    from gui.fitCommands.calc.booster.add import CalcAddBoosterCommand
  File "/usr/local/Pyfa/gui/fitCommands/calc/booster/add.py", line 4, in <module>
    from service.fit import Fit
  File "/usr/local/Pyfa/service/fit.py", line 35, in <module>
    from service.character import Character
  File "/usr/local/Pyfa/service/character.py", line 36, in <module>
    from service.esi import Esi
  File "/usr/local/Pyfa/service/esi.py", line 14, in <module>
    from service.esiAccess import APIException, GenericSsoError
  File "/usr/local/Pyfa/service/esiAccess.py", line 12, in <module>
    from jose import jwt
  File "/home/doodidan/.local/lib/python3.10/site-packages/jose/jwt.py", line 5, in <module>
    from collections import Mapping
ImportError: cannot import name 'Mapping' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
```

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):

Ubuntu 22.04
Python 3.10

### Other relevant information:

I use self builded wxPython

I tried to update `python-jose` but other errors appear

**Comment by blitzmann** [comment]
Duplicate issue #2391, There are some breaking changes with 3.10 and wxPython. Recommend 3.7 / 3.8 with wxPython 4.0.6 specifically, or use the AppImage for now

@DarkFenX  has been working on a branch when time permits to update dependencies, which would also presumably fix compatibility issues with 3.10

**Comment by GoshaEgorian** [comment]
@blitzmann Where I can find AppImage for Ubuntu? I didn't find it last time I set up Pyfa
